src/helpers/xml-builder.js

```diff
@@ -712,49 +712,70 @@ const buildRun = async (vNode, attributes, docxDocumentInstance) => {
     const textFragment = buildTextElement(vNode.text);
     runFragment.import(textFragment);
   } else if (attributes && attributes.type === 'picture') {
-    let response = null;
 
-    const base64Uri = decodeURIComponent(vNode.properties.src);
-    if (base64Uri) {
-      response = docxDocumentInstance.createMediaFile(base64Uri);
+    const imageSource = vNode.properties.src;
```

Comment:
Unable to get this working outside of html-to-docx - see comments in chat

Reply:
Not sure what you mean here